Rakefile

```diff
@@ -50,9 +50,22 @@ end
 task :nylas_build do
   Jeweler::Tasks.new do |gem|
     setup_nylas_gem(gem)
+    gem.platform = 'java'
```

Comment:
I've never published a JRuby gem before --- Does JRuby use a different rubygems repository? Also, won't your changes prevent us from publishing the MRI version of the gem?